src/mixins/observable.mixin.js

```diff
@@ -46,6 +46,27 @@
     return this;
   }
 
+  function _once(eventName, handler) {
+    var _handler = function () {
+      handler();
```

Comment:
@ShaMan123 in this case handler is not bound and is not getting the arguments.
I think this should  somethng like this:

```js
var _handler = function () {
   handler.apply(this, arguments)
   this.off(...)
}.bind(this)
```

handler needs to be bound when executed and needs to receive the arguments.
Can you verify and update the tests?

Reply:
@ShaMan123
bind handler
c63e746